sklearn/feature_extraction/image.py

```diff
@@ -261,8 +261,8 @@ def _compute_n_patches(i_h, i_w, p_h, p_w, max_patches=None):
     max_patches : int or float, default=None
         The maximum number of patches to extract. If max_patches is a float
         between 0 and 1, it is taken to be a proportion of the total number
-        of patches.
-        If max_patches=None then the total number of patches will be extracted.
+        of patches. If `max_patches` is None it corresponds to the total number
```

Comment:
nit: I can't really explain why but I think I prefer "If `max_patches` is None all possible patches are extracted.". I find "total number of patches" a bit weird.